src/components/component-playground.jsx

```diff
@@ -109,6 +113,7 @@ module.exports = React.createClass({
     preview: function() {
       var params = {
         component: this.constructor.getSelectedComponentClass(this.props),
+        ref: ((ref) => this.refs.preview = ref),
```

Comment:
They may be regarded as legacy, but they still work. You should investigate a bit more why strings don't work before using this workaround. If you still can't figure it out, add some comments here explaining the situation.